opensearch_secure_integration_test.go

```diff
@@ -37,11 +40,22 @@ import (
 )
 
 func getSecuredClient() (*opensearchapi.Client, error) {
+	version := os.Getenv("OPENSEARCH_VERSION")
+	parts := strings.Split(version, ".")
```

Comment:
We have a [ParseVersion function](https://github.com/opensearch-project/opensearch-go/blob/main/opensearch.go#L239) that can be used.